yaks/lib/yaks/util.rb

```diff
@@ -7,26 +7,26 @@ module Util
 
     def underscore(str)
       str.gsub(/::/, '/')
-        .gsub(/(?<!^|\/)([A-Z])(?=[a-z$])|(?<=[a-z])([A-Z])/, '_\1\2')
+        .gsub(%r{(?<!^|/)([A-Z])(?=[a-z$])|(?<=[a-z])([A-Z])}, '_\1\2')
         .tr("-", "_")
         .downcase
     end
 
     def camelize(str)
-      str.gsub(/\/(.?)/)     { "::#{ $1.upcase }" }
-        .gsub!(/(?:^|_)(.)/) { $1.upcase          }
+      str.gsub(%r{/(.?)})    { "::#{ Regexp.last_match(1).upcase }" }
+        .gsub!(/(?:^|_)(.)/) { Regexp.last_match(1).upcase          }
```

Comment:
I just recalled: I think back in some point in time `$1` was not thread-safe (while `Regexp.last_match` was) – but I believe that’s no longer the case anyway. :)